packages/js/product-editor/src/blocks/product-fields/inventory-email/edit.tsx

```diff
@@ -52,10 +54,13 @@ export function Edit( {
 		`low_stock_amount-${ clientId }`,
```

Comment:
Given the above comment and the usefullness of the the `clientId`, it may be worth stripping the prefix and only using the `clientId`, or provide the `clientId` as a third parameter.
cc: @mdperez86 may have some thoughts on this.

Reply:
Well this is by far out of the scope of this PR. But I have to say that it does not matter the validatorId you picked or the way you build it, the only restriction is that it has to be unique. And I do not agree of having a third parameter for the validator registration function since the validations are intentionally agnostic of Blocks.